subprojects/xta/xta/src/main/java/hu/bme/mit/theta/xta/utils/ClockType.java

```diff
@@ -33,4 +34,8 @@ public String toString() {
         return "Clock";
     }
 
+    @Override
+    public DomainSize getDomainSize() {
+        return DomainSize.ONE;
```

Comment:
I am not sure if this is correct.

Reply:
As I browsed through the code, I think the `ClockType` builds on `RatType`, so this should be `INFINITY`,